<launch>
  <arg name="input_route" default="input_route"/>
  <arg name="input_occupancy_grid" default="input_occupancy_grid"/>
  <arg name="input_scenario" default="input_scenario"/>
  <arg name="input_odometry" default="/localization/kinematic_state"/>
  <arg name="output_trajectory" default="output_trajectory"/>
  <arg name="is_completed" default="is_completed"/>

  <arg name="param_file" default="$(find-pkg-share freespace_planner)/config/freespace_planner.param.yaml"/>

  <!-- vehicle info -->
  <arg name="vehicle_info_param_file" default="$(find-pkg-share autoware_vehicle_info_utils)/config/vehicle_info.param.yaml"/>

  <node pkg="autoware_freespace_planner" exec="freespace_planner" name="freespace_planner" output="screen">
    <remap from="~/input/route" to="$(var input_route)"/>
    <remap from="~/input/occupancy_grid" to="$(var input_occupancy_grid)"/>
    <remap from="~/input/scenario" to="$(var input_scenario)"/>
    <remap from="~/input/odometry" to="$(var input_odometry)"/>
    <remap from="~/output/trajectory" to="$(var output_trajectory)"/>
    <remap from="is_completed" to="$(var is_completed)"/>
    <param from="$(var param_file)"/>
    <param from="$(var vehicle_info_param_file)"/>
  </node>
</launch>
